### PR TITLE
A Bit Of Repo Cleaning

### PR DESCRIPTION
**Pull request by PythooonUser**
## Summary
This PR attempts to clean the repo a bit with the following:
- adding several IDE config files regarding the Java project. I feel like these could be universal. Contrary, we can make these ignored
- updating to `1.8` version
- updating `packaged_files.txt` (see #188, we might want to add it to `.gitignore` lateron)

### CODE REVIEW

File: .settings/org.eclipse.buildship.core.prefs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+arguments=
+auto.sync=false
+build.scans.enabled=false
+connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)
+connection.project.dir=
+eclipse.preferences.version=1
+gradle.user.home=
+java.home=C\:/Program Files/AdoptOpenJDK/jdk-14.0.2.12-hotspot
```
Comment (joshuaskelly) [inline .settings/org.eclipse.buildship.core.prefs:8]: This is an machine specific setting. Is there anyway to use JAVA_HOME environment variable?

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,9 @@
+## IDE
+*.iml
+.idea/*
+!/.idea/runConfigurations
+.vscode/
```
Comment (joshuaskelly) [inline .gitignore:5]: Should we include the .vscode directory? It would be nice to have the file associations + JSON schema already hooked up.

**Review by joshuaskelly — COMMENTED** [review]
Some things to chat about. 😊

File: .settings/org.eclipse.buildship.core.prefs (hunk 1 above)
Comment (PythooonUser) [reply .settings/org.eclipse.buildship.core.prefs:8]: Oh, didn't see that. Good catch! Now that I'm looking at this, all of these parameters seem to be dependent on the local setup, so maybe that file could be ignored instead. On Windows it is a very weird Java setup, so not sure about JAVA_HOME.

File: .gitignore (hunk 2 above)
Comment (PythooonUser) [reply .gitignore:5]: This is driven by `settings.json`. Here, I also have some other stuff defined by the Java extension. Same for `launch.json`. How about commiting a *light* version as `settings.json.dist` or something along those lines?

File: .settings/org.eclipse.buildship.core.prefs (hunk 1 above)
Comment (PythooonUser) [reply .settings/org.eclipse.buildship.core.prefs:8]: I don't have JAVA_HOME set as an environment variable, but rather as some config in VS Code itself.

**Comment by PythooonUser** [comment]
Has been taken care of with #194. Closing.

**Pull request closed without merging**